Hunting Queries/OracleDatabaseAudit/OracleDBAuditActionsByIp.yaml

```diff
@@ -0,0 +1,30 @@
+id: b43e074f-b630-4a16-80e5-138f4b6b64fb
+name: OracleDBAudit - Action by Ip
+description: |
+  'Query searches sources from which DbActions were made.'
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+tactics:
+  - InitialAccess
+  - DefenseEvasion
+  - Collection
+  - Impact
+query: |
+  let lbtime = 24h;
+  OracleDatabaseAuditEvent
+  | where TimeGenerated > ago(lbtime)
+  | where isnotempty(DbAction)
+  | where isnotempty(SrcIpAddr)
+  | where DbAction in~ ('SELECT', 'DROP', 'INSERT')
+  | extend TableName = replace(@'[,\(\)]', '', extract(@'(?i)SELECT(.*?)FROM\s(.*?)\s', 2, Action))
+  | where isnotempty(TableName)
+  | where TableName !in ('SELECT', 'select')

```

Comment:
use `| where TableName !~ 'select'` <-- case insensitive match

Reply:
updated

Reply:
looks like this one did not get updated

Reply:
fixed